client/views/msg_condensed.tpl

```diff
@@ -1,5 +1,5 @@
-<div class="msg {{type}} closed" data-time="{{time}}">
-	<span class="time hide-text">{{tz time}}</span>
+<div class="msg condensed closed">
+	<span class="time">00:00</span>
```

Comment:
Yes, I'm stupid. Goddammit, I was so hoping this was the ~right~ _better_ thing... So should I just leave actual time as is, for now? :/